client/src/components/App.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import TopBar from './TopBar.jsx'
+import VideoPLayer from './VideoPLayer.jsx'
+import StreamDetail from './StreamDetail.jsx'
+import '../styles.css';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      streamerId: 3,
```

Comment:
Should this be null or underfined instead of 3?

Reply:
I will run some test to see if it throws any error when changed.